### ISSUE TITLE
Where are the files?

### ISSUE THREAD

**Issue by dixinyinli2** [trainingl/STG4Traffic#6]
Excuse me, I found this in your code, but I can not find the files. I want to know where are the files. Thanks a lot!
`dataset_dir = "../data/PEMS-BAY/processed/"
graph_pkl = "../data/PEMS-BAY/processed/adj_mx.pkl"`

**Comment by WaldenBlues** [comment]
The project has links about PEMS-Bay dataset but not involved in the repo files. As for missing files or wrong dir problem, (If dataset exists) some may needed to be processed by generate_data.py (the reason for PEMS-Bay/processed/ dir existed)

**Comment by trainingl** [comment]
> Excuse me, I found this in your code, but I can not find the files. I want to know where are the files. Thanks a lot! `dataset_dir = "../data/PEMS-BAY/processed/" graph_pkl = "../data/PEMS-BAY/processed/adj_mx.pkl"`

Yeah, you need to get the raw data source like METR. Then preprocess the data by using TrafficSpeed/lib/generate_data.py and TrafficSpeed/lib/generate_adj_mx.py files.

**Comment by trainingl** [comment]
> Excuse me, I found this in your code, but I can not find the files. I want to know where are the files. Thanks a lot! `dataset_dir = "../data/PEMS-BAY/processed/" graph_pkl = "../data/PEMS-BAY/processed/adj_mx.pkl"`

If you still have questions, it would be more convenient to contact me by email.

**Comment by dixinyinli2** [comment]
> > Excuse me, I found this in your code, but I can not find the files. I want to know where are the files. Thanks a lot! `dataset_dir = "../data/PEMS-BAY/processed/" graph_pkl = "../data/PEMS-BAY/processed/adj_mx.pkl"`
> 
> Yeah, you need to get the raw data source like METR. Then preprocess the data by using TrafficSpeed/lib/generate_data.py and TrafficSpeed/lib/generate_adj_mx.py files.

Thank you, I will try it!
